resources/assets/less/bem/chat-form.less

```diff
@@ -4,19 +4,20 @@
 .chat-form {
   --input-size: @font-size--title-small-3;
 
+  display: flex;
+  flex-direction: column;
   flex: 1;
   background: hsl(var(--hsl-b4));
 
   &__button-bar {
```

Comment:
the bar should probably be get rid of entirely so the margins are uniform

![](https://s.myconan.tk/2023-01/firefox_2023-01-16_21-09-45.png)